appstudio-utils/util-scripts/lib/fetch/cluster.sh

```diff
@@ -16,5 +16,6 @@ k8s-save-data() {
 # Placeholder. In future it will likely come from
 # an EnterpriseContractPolicy crd, see HACBS-244
 save-policy-config() {
-  echo '{"non_blocking_checks":["not_useful"]}' | jq > $( json-data-file config policy )
+  # Note: the namespace the task is running in needs to have the ec-policy ConfigMap
+  { oc get configmap ec-policy -o go-template='{{index .data "policy.json"}}' 2>/dev/null || echo '{"non_blocking_checks":["not_useful"]}'; } | jq > "$( json-data-file config policy )"
```

Comment:
Nitpick:
* line is a bit too long
* Just an idea: Could make it nicer by splitting it up maybe, e.g.

```
real-policy() {
  ...
}

dummy-policy() {
  ...
}

save-policy-config()
  { real-policy || dummy-policy } | jq ...
```

Though it might hit some `set -e` issues, not sure. On the plus side the shellspec will let you know right away.

Reply:
Much nicer